packages/cli/README.md

```diff
@@ -382,92 +382,101 @@ OPTIONS
   -c, --ceramic=ceramic  Ceramic API URL
 ```
 
-### `glaze stream:commits STREAMID`
+### `glaze stream:commits STREAMID SYNC_OPTION`
```

Comment:
I feel like the sync option should be a --flag, rather than a top-level positional argument